index.ts

```diff
@@ -0,0 +1,54 @@
+const prompt = require("prompt-sync")({ sigint: true });
+const upperValue: string | null = prompt("What value would you like FizzBuzz to go up to? ");
+console.log(`We go up to ${upperValue}`);
+
+function divisibleBy(num: number, divisor: number): boolean { return num % divisor == 0 };
+
+const key: {[index: number]: Function } = {
```

Comment:
what does key mean? can we name this something more informative maybe?

Reply:
Also can we be more specific with the type of function? it appears to always be `(answer: string[]) => string[]`